src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -111,74 +154,164 @@ internal abstract class AbstractAddParameterCodeFixProvider<
             }
 
             var arguments = (SeparatedSyntaxList<TArgumentSyntax>)syntaxFacts.GetArgumentsOfObjectCreationExpression(objectCreation);
+            var methodCandidates = type.InstanceConstructors;
+            var typeArguments = (SeparatedSyntaxList<TTypeSyntax>)syntaxFacts.GetTypeArgumentsOfObjectCreationExpression(objectCreation);
+            var candidates = GetArgumentInsertPositionAndTypeArgumentForMethodCandidates(semanticModel, syntaxFacts, methodCandidates, arguments, argumentOpt, typeArguments);
+            RegisterFixForMethodOverloads(context, arguments, candidates);
+        }
 
-            var comparer = syntaxFacts.StringComparer;
-            var constructorsAndArgumentToAdd = ArrayBuilder<(IMethodSymbol constructor, TArgumentSyntax argument, int index)>.GetInstance();
+        private void RegisterFixForMethodOverloads(CodeFixContext context, SeparatedSyntaxList<TArgumentSyntax> arguments, ImmutableArray<(IMethodSymbol method, TArgumentSyntax argument, int index, TTypeSyntax typeArgument)> methodsAndArgumentToAdd)
```

Comment:
wrap parameter lists longer than github width plz.

Reply:
consider renaming methodsAndArgumentToAdd to insertionData.

consider using a real named type here as excessively long tuples end up being hard to read.